src/ScrollyVideo.jsx

```diff
@@ -1,44 +1,79 @@
-import React, { useEffect, useRef, useState } from 'react';
+import React, { useEffect, useRef } from 'react';
```

Comment:
per my comment above, it would be great to remove the changes for files other than this one to keep the demos clean and barebones.